server/models/db_entities/db_item.py

```
@@ -30,13 +30,15 @@ def __str__(self):
 
 
 ##### Missing: Blobs handling
+# Returns: DBItem instance, or None on failure.
 def get_item(item_id):
     if(type(item_id) != int):  # MAYBE WE ALSO PREVENT USER FROM INPUTING ANYTHING OTHER THAN A NUMBER (in the  web forms + in the HTTP requests).
-        return("item_id must be int")
+        return("item_id must be int!")
```

Instead of returning a String here, I would suggest throwing a custom error. 
You can create a file somewhere called exceptions.py or something like that and then define your own custom errors as you see fit, for example: 

class InvalidDBRecordIDException(Exception):
      pass

then you can raise this error at this point, and you can catch it somewhere else in your program and handle it.

---

Good notice!
I was actually aware of this when I wrote it, I just kept it this way for now because this error handling deserves its own Issue and PR, plus I was sure we'll face it again in the next step when we configure the server to use the DBItem class.
I kept it this way for now.